content/general/concepts/unix/unix.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'Unix'
+Description: 'Unix is a family of operating systems that are all derived from the original AT&T Unix developed at Bell Labs in 1969.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Developer Tools'
+  - 'Operating Systems'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/code-foundations'
+---
+
+Unix is a family of operating systems that are all derived from the original AT&T Unix developed at Bell Labs in 1969. Unix is a multitasking, multiuser operating system that is characterized by the implementation of many small modular software tools that can be combined using a scripting language known as the [Unix shell](https://en.wikipedia.org/wiki/Unix_shell). Unix and Unix-like operating systems are currently used widely on servers, workstations and mobile devices.
+
+Originally developed for internal use at Bell labs, in the late 1970s AT&T licensed Unix to a variety of outside parties, leading to a number of academic and commercial Unix variants. These include:
+
+- BSD: From the University of California, Berkley.
+- SunOS/Solaris: From Sun Microsystems.
+- AIX: From IBM
+
+In 1991, [Linus Torvalds](https://www.codecademy.com/resources/docs/general/linus-torvalds) released the [open source](https://www.codecademy.com/resources/docs/general/open-source) [Linux](https://www.codecademy.com/resources/docs/general/linux) kernel as a Unix-like free alternative to proprietary Unix systems. It had since exploded into dozens, if not hundreds of different variant distributions.
```

Comment:
```suggestion
## Timeline

In 1991, [Linus Torvalds](https://www.codecademy.com/resources/docs/general/linus-torvalds) released the [open source](https://www.codecademy.com/resources/docs/general/open-source) [Linux](https://www.codecademy.com/resources/docs/general/linux) kernel as a Unix-like free alternative to proprietary Unix systems. It had since exploded into dozens, if not hundreds of different variant distributions.
```